README.md

```
@@ -237,10 +239,13 @@ This is what handles the accessory logic like syncing states between HomeKit and
 ### 4.2 Setup
 
 > [!IMPORTANT]
-> Only PN532 over SPI(`pn532_spi` component) is supported at the moment due to required modifications that haven't been ported to other protocols or chips
+> Only PN532 over SPI(`pn532_spi` component) or the PN532 over HSU(`pn532_uart` component) are supported at the moment due to required modifications that haven't been ported to other protocols or chips
 
```

markdownlint-cli2 (0.21.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)